src/utils/testing.utils.ts

```diff
@@ -21,6 +21,7 @@ export function fakeResponse(overrides?: Partial<Response>): Response {
   res.send = jest.fn().mockImplementation(() => res)
   res.end = jest.fn().mockImplementation(() => res)
   res.cookie = jest.fn().mockImplementation((n: string, v: string, o: CookieOptions) => res)
+  res.clearCookie = jest.fn().mockImplementation((name: string, options: CookieOptions) => res)
```

Comment:
```suggestion
  res.clearCookie = jest.fn().mockImplementation((name: string, options?: CookieOptions) => res)
```

That's my b. Should have mentioned that options are an optional (lmao) parameter. So this is just bringing this more in line with how clearCookie actually works.